src/BloomExe/web/ReadersHandler.cs

```diff
@@ -222,7 +222,7 @@ private static void MakeLetterAndWordList(string jsonSettings, string allWords)
 
 			// format the output
 			var sb = new StringBuilder();
-			sb.AppendLineFormat("Letter and word list for making decodable readers in {0}", CurrentBook.CollectionSettings.Language1Name);
+			sb.AppendLineFormat("The following is a generated report of the decodable stages for {0}.  You can make any changes you want to this file, but Bloom will not notice your changes.  It is just a report.", CurrentBook.CollectionSettings.Language1Name);
```

Comment:
Would you just go ahead and do it? This is the competition area of the program, and they've already complained about missing localization, so I'm pretty sure we should be localizing this unless you know of some other point in the process where it might happen.